libraries/chain/include/eosio/chain/backing_store/db_combined.hpp

```diff
@@ -209,76 +194,128 @@ class rocksdb_whole_db_table_collector {
                               b1::chain_kv::bytes::const_iterator key_end, const char *value,
                               std::size_t value_size) {
-      uint64_t primary_key;
-      EOS_ASSERT(b1::chain_kv::extract_key(remaining, key_end, primary_key), bad_composite_key_exception,
-                 "DB intrinsic key-value store composite key is malformed");
-      backing_store::payer_payload pp(value, value_size);
-      receiver_.add_row(primary_index_view{primary_key, pp.payer, {std::string{pp.value, pp.value + pp.value_size}}});
+      if (!is_table_only()) {
+         uint64_t primary_key;
+         EOS_ASSERT(b1::chain_kv::extract_key(remaining, key_end, primary_key), bad_composite_key_exception,
+                    "DB intrinsic key-value store composite key is malformed");
+         receiver_.add_row(primary_index_view::create(primary_key, value, value_size));
+      }
       ++primary_count_;
    }
 
    template<typename IndexType>
    void extract_secondary_index(b1::chain_kv::bytes::const_iterator remaining,
                                 b1::chain_kv::bytes::const_iterator key_end, const char *value,
                                 std::size_t value_size) {
-      IndexType secondary_key;
-      uint64_t primary_key;
-      EOS_ASSERT(b1::chain_kv::extract_key(remaining, key_end, secondary_key), bad_composite_key_exception,
-                 "DB intrinsic key-value store composite key is malformed");
-      EOS_ASSERT(b1::chain_kv::extract_key(remaining, key_end, primary_key), bad_composite_key_exception,
-                 "DB intrinsic key-value store composite key is malformed");
-      backing_store::payer_payload pp(value, value_size);
-      receiver_.add_row(secondary_index_view<IndexType>{primary_key, pp.payer, secondary_key});
+      if (!is_table_only()) {
+         IndexType secondary_key;
+         uint64_t primary_key;
+         EOS_ASSERT(b1::chain_kv::extract_key(remaining, key_end, secondary_key), bad_composite_key_exception,
+                    "DB intrinsic key-value store composite key is malformed");
+         EOS_ASSERT(b1::chain_kv::extract_key(remaining, key_end, primary_key), bad_composite_key_exception,
+                    "DB intrinsic key-value store composite key is malformed");
+         backing_store::payer_payload pp(value, value_size);
+         receiver_.add_row(secondary_index_view<IndexType>{primary_key, pp.payer, secondary_key});
+      }
       ++secondary_count<IndexType>();
    }
 
    bool operator()(uint64_t contract, const char *key, std::size_t key_size,
                    const char *value, std::size_t value_size) {
-      if (!keep_processing_()) {
-         return false;
-      }
       b1::chain_kv::bytes composite_key(key, key + key_size);
       auto[scope, table, remaining, type] = backing_store::db_key_value_format::get_prefix_thru_key_type(
             composite_key);
 
+      const name code = name{contract};
```

Comment:
Add a comment before "bool operator()(..." method:
//                DB Intrinsic Keys
// PREFIX:
// db intrinsic keys are all prefaced with the following information:
// | contract | scope | table | type of key |
// so when iterating through the database, all keys for a given contract/scope/table will be processed together
//
// TYPE OF KEY:
// the "type of key" is an enum which ends up creating the following order for the type of keys:
// primary key
// primary to secondary key (not present in chainbase structure since it can have more than one index on the same structure -- these are ignored for this class's processing)
// uint64_t secondary key
// uint128_t secondary key
// key256_t secondary key
// float64_t secondary key
// float128_t secondary key
// table key (indicates end of table and payer)
//
// KEY PROCESSING (key_context):
// keys may be processed as:
// individual key (standalone)
//     -- this can be any key - primary, secondary, or table
// table only (table_only, table_only_reversed)
//     -- all keys for a table are passed in and processed only to report a valid table_id_object_view, if the table is 
//         passed in table_only, the table is determined to be complete when the table key is received, for 
//         table_only_reversed all of the table_id_object_view data, other than the count, is determined when the
//         table key is received and the count is determined when the table is determined to be complete (either by a
//         new key being received that is for a different table, or the complete() method is called.
//     -- only the the table_id_object_view is reported, none of the other key's objects are reported
// complete (complete, complete_reverse)
//     -- similar to table only, except all of the objects for the keys in the table are also reported

Reply:
Added.